synapse/lib/ast.py

```diff
@@ -699,7 +699,92 @@ async def todogenr():
                 yield node, path
 
 class Oper(AstNode):
-    pass
+
+    async def yieldFromValu(self, runt, valu, vkid):
```

Comment:
Moved this up from `YieldValu`, unchanged other than adding a `vkid` arg to specify where error highlighting should go.